package-lock.json

```diff
@@ -500,7 +500,7 @@
     },
     "commander": {
       "version": "2.15.1",
-      "resolved": "https://registry.npmjs.org/commander/-/commander-2.15.1.tgz",
+      "resolved": "http://registry.npmjs.org/commander/-/commander-2.15.1.tgz",
```

Comment:
Could you try to revert this bunch of changes to `package-lock.json`.
There are no version updates however a lot of unneeded `https` -> `http`, and removed `"optional": true` lines

Reply:
done.
I think... 👍